sqlitecluster/SQLite.cpp

```diff
@@ -73,8 +73,14 @@ SQLite::SQLite(const string& filename, int cacheSize, int autoCheckpoint, int ma
     // any tables to be effective.
     SASSERT(!SQuery(_db, "new file format for DESC indexes", "PRAGMA legacy_file_format = OFF"));
 
+    // Compare to see if "false" wasn't passed meaning the -synchronous flag was set and run the query
+    if (!synchronous.empty()) {
+        SASSERT(!SQuery(_db, "setting custom synchronous commits", "PRAGMA synchronous = " + synchronous  + ";"));
```

Comment:
Should probably wrap `synchronous` in `SQ()`. Also, this doesn't log anything when it works.